language-support/java/bindings/src/main/java/com/daml/ledger/javaapi/data/codegen/json/JsonLfWriter.java

```diff
@@ -40,4 +47,46 @@ void writeComma() throws IOException {
   void write(String value) throws IOException {
     writer.write(value);
   }
+
+  void writeNumeric(BigDecimal num) throws IOException {
+    if (options.encodeNumericAsString) {
+      writer.write(quoted(num));
+    } else {
+      writer.write(num.toString());
+    }
+  }
+
+  void writeInt64(Long num) throws IOException {
+    if (options.encodeInt64AsString) {
+      writer.write(quoted(num));
+    } else {
+      writer.write(num.toString());
+    }
+  }
+
+  private String quoted(Object val) {
+    return "\"" + val.toString() + "\"";
+  }
+
+  public static Options opts() {
+    return new Options(true, true); // TODO: read these from sysprops?
```

Comment:
This is where we set the default behaviour.

Reply:
do we have precedent for this? Or should we aim to have less behavior and thus less surface area wrt backwards compatibility? your call.

Reply:
The precedent is this
https://docs.daml.com/json-api/lf-value-specification.html#codec-library
where we had configuration for `encodeDecimalAsString` and `encodeInt64AsString`. 
I've used `encodeNumericAsString` here instead of using the term `Decimal` as IIUC that was the name for the old `Numeric 10` before we had `Numeric`.